recipes/viva-engage/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
since you are renaming an existing recipe, please add the `alias` section in here

Reply:
added in 9874ac2a7040ba9ca895c7af9e4b715609dda836